src/ukify/ukify.py

```diff
@@ -1394,9 +1399,30 @@ def config_example(self) -> tuple[Optional[str], Optional[str], Optional[str]]:
 
 def apply_config(namespace, filename=None):
     if filename is None:
-        filename = namespace.config
+        if namespace.config:
+            # Config file specified, turn it into a pathlib.Path
+            filename = pathlib.Path(namespace.config)
+        else:
+            # If there is no config file provided, try to look for default configs and load the first found.
+            for config_dir in DEFAULT_CONFIG_DIRS:
+                config_path = pathlib.Path(config_dir).joinpath(DEFAULT_CONFIG_FILE)
+                if config_path.is_file():
+                    print(f'No config file specified but found config file: {config_path}')
+                    filename = config_path
```

Comment:
Please drop the `config_path` variable and just use `filename` as the iterator. If the loop doesn't find anything, we return from the function immediately, so no need to keep those separate.